BookStore/database/migrations/2021_09_24_125630_create_orderitems_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrderitemsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orderitems', function (Blueprint $table) {
+            $table->id();
+            $table->integer('order_id');
+            $table->integer('book_id');
```

Comment:
Create foreign key for orders and books:
`$table->foreignId('order_id');`
`$table->foreignId('book_id');`
`$table->foreign('book_id')->references('id')->on('books');`
`$table->foreign('order_id')->references('id')->on('orders');`